jumpscale/core/executors/remote.py

```diff
@@ -35,6 +35,17 @@ def execute(cmd, command_ctx, connection_ctx):
 
 
 class RemoteExecutor:
+    """Executor remote allows executing commands within specific env on the local machine. using the executor framework you can retrieve the stdout, stderr, and the return code as well.
```

Comment:
Maybe using "remote machine" here is better?

```suggestion
    """Remote executor allows executing commands within specific env on the remtote machine. using the executor framework you can retrieve the stdout, stderr, and the return code as well.
```

Reply:
done

Reply:
done